.github/workflows/ci.yaml

```diff
@@ -57,6 +59,39 @@ jobs:
       - name: Lint
         run: make lint
 
+  check-nuctl-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          cache: true
+          go-version-file: go.mod
+
+      - name: Generate Nuctl docs
+        run: make generate-nuctl-docs
+
+      - name: Get Makefile Variable
+        id: get-nuctl-docs-path
+        run: echo "NUCTL_DOCUMENTATION_PATH=$(make print-nuctl-docs-path | tail -n 1)" >> $GITHUB_ENV
```

Comment:
What is this for? it is unused later

Reply:
@TomerShor right, fixed it. This step is needed to use Makefile's nuctl docs path as the only source of truth